app/database/schema.prisma

```diff
@@ -344,6 +344,8 @@ model Location {
   description String?
   address     String?
 
+  imageUrl String?
+
   image   Image?  @relation(fields: [imageId], references: [id])
```

Comment:
We have to think how to do this tbh. As we cannot remove it with a migration because our open source users will loose their data. Maybe we have to leave it and warm everyone who uses shelf via the repository discussions that this will be removed with a migration and they have to run the scripts first. IDK we have to write down a step-by-step migration plan for our users.